.changeset/good-pillows-do.md

```diff
@@ -0,0 +1,6 @@
+---
+'@guardian/source-foundations': major
+'@guardian/source-react-components': major
```

Comment:
Just regarding this, we’re assuming react-components is getting a bump because foundations is a peerDependency

Reply:
If this is the case: wouldn’t we also expect react-components-development-kitchen to receive a bump also? 🤔

Reply:
Yes! In the changesets list in the box above, it shows a major bump to all packages... 
![image](https://user-images.githubusercontent.com/15648334/169260324-6dd6ba5a-b71a-42cf-bf71-fc49c48421fb.png)

Reply:
Ohhh cool! I'm probably misunderstand what the `.changeset` markdown file is for, I'll do some more reading 😅

Reply:
So maybe we can remove `'@guardian/source-react-components': major` from this file? Assuming changesets is just "aware" that it needs to be bumped, we don't need to specify it in the changeset file?

Reply:
My suggestion would be to remove this from here, and then I think we're good to merge. What do you think?

Reply:
Aha. Changesets is picking up changes to `@guardian/source-react-components` because this package has changed vs the [baseBranch](https://github.com/guardian/source/blob/858ebd7c52feb4db4a0971ffe52f972ee16a9505/.changeset/config.json#L7)